Command/DeleteOrphanIndexesCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace EMS\CoreBundle\Command;
+
+use Elasticsearch\Client;
+use EMS\CoreBundle\Service\AliasService;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Elasticsearch\Common\Exceptions\Missing404Exception;
+
+class DeleteOrphanIndexesCommand extends EmsCommand
+{
+    /** @var LoggerInterface */
+    protected $logger;
+    /** @var Client */
+    protected $client;
+    /** @var AliasService */
+    protected $aliasService;
+
+    public function __construct(LoggerInterface $logger, Client $client, AliasService $aliasService)
+    {
+        $this->logger = $logger;
+        $this->client = $client;
+        $this->aliasService = $aliasService;
+        parent::__construct($logger, $client);
+    }
+
+    protected function configure()
+    {
+        $this->setName('ems:delete:orphans')
+            ->setDescription('Removes all orphan indexes');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $this->aliasService->build();
+        foreach ($this->aliasService->getOrphanIndexes() as $index) {
+            $this->deleteOrphanIndex($index);
+        }
+    }
+
+    private function deleteOrphanIndex(array $index)
```

Comment:
This function does exactly the same as deleteOrphansIndexesAction in the Controller. Maybe it's time to refactor this in a IndexService ? 

Also this would improve DI : $client = $this->getElasticsearch(); is deprecated (use DI instead)